ChefKnivesBot/Handlers/Comments/MakerPostCommentHandler.cs

```diff
@@ -39,7 +40,7 @@ public async Task<bool> Process(BaseController baseController, Func<string, Task
             // Check if the link flair matches the maker post flait and that the author is not a moderator
             if (linkFlairId != null && linkFlairId.Equals(_makerPostFlair.Id))
             {
-                if (comment.Removed || comment.Listing.Approved || comment.Author.Equals(_service.Account.Me.Name))
+                if (comment.Removed || comment.Listing.Approved || CommentAuthorAutoApproved(comment.Author.ToLower()))
```

Comment:
I believe we want both conditions, so that the bot doesn't try to remove its own comments.